src/pykeen/trackers/wandb.py

```diff
@@ -28,16 +28,17 @@ class WANDBResultTracker(ResultTracker):
     def __init__(
         self,
         project: str,
-        experiment: Optional[str] = None,
         offline: bool = False,
         **kwargs,
     ):
         """Initialize result tracking via WANDB.
 
         :param project:
             project name your WANDB login has access to.
-        :param experiment:
-            The experiment name to appear on the website. If not given, WANDB will generate a random name.
+        :param offline:
```

Comment:
this parameter was not document before.

From https://docs.wandb.ai/guides/track/launch it looks like the environment variable should not be set to `dryrun` but rather `offline`. I am not sure about the use case here either: maybe we should not set any environment variables from within pykeen, but rather require the user to actively do it. Everything else can be passed via the `kwargs`, which is directly forwarded to `wandb.init`.

Reply:
eww I wish we could set them directly and not with an environment variable.